volatility3/framework/plugins/windows/info.py

```diff
@@ -7,11 +7,28 @@
 
 from volatility3.framework import constants, interfaces, layers, symbols
 from volatility3.framework.configuration import requirements
-from volatility3.framework.interfaces import plugins
+from volatility3.framework.interfaces import plugins, configuration
 from volatility3.framework.renderers import TreeGrid
 from volatility3.framework.symbols import intermed
 from volatility3.framework.symbols.windows import extensions
+from volatility3.framework.layers import physical
+import struct
```

Comment:
I recommend that arrange the modules in alphabetical order.
And recommended that the local package, not the module (API) created by Volatility, be directed to the top.

For Example:
```
import struct

from volatility3.framework.interfaces import configuration, plugins # Point
from volatility3.framework.layers import physical # Point
from volatility3.framework.renderers import TreeGrid
from volatility3.framework.symbols import intermed
from volatility3.framework.symbols.windows import extensions
```

Reply:
I have moved the import

Reply:
Thank you for reflecting my opinion!